From b04da73d7ff4e647960ed9e9bf87cf100972db47 Mon Sep 17 00:00:00 2001
From: Zhang Zhijie <zhangzj@rock-chips.com>
Date: Mon, 5 Jun 2017 16:13:23 +0800
Subject: [PATCH] OP-TEE: remove macro SWITCH_CPU0_DEBUG

Now OP-TEE OS can run in muti-core state,
so don't need switch to cpu0 anymore.'

Change-Id: I4f61120250823d6e2e13e2edeee58c26a184b7d2
Signed-off-by: Zhang Zhijie <zhangzj@rock-chips.com>
---
 security/optee_linuxdriver/armtz/tee_tz_drv.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/security/optee_linuxdriver/armtz/tee_tz_drv.c b/security/optee_linuxdriver/armtz/tee_tz_drv.c
index 824639489a38..846c249a1196 100644
--- a/security/optee_linuxdriver/armtz/tee_tz_drv.c
+++ b/security/optee_linuxdriver/armtz/tee_tz_drv.c
@@ -43,7 +43,7 @@
 #undef CONFIG_OUTER_CACHE
 #endif
 
-#define SWITCH_CPU0_DEBUG
+/* #define SWITCH_CPU0_DEBUG */
 
 #define _TEE_TZ_NAME "armtz"
 #define DEV (ptee->tee->dev)
-- 
2.35.3

